src/main/java/console/output/ConsolePrinter.java

```diff
@@ -0,0 +1,57 @@
+package console.output;
```

Comment:
Is this package name viable? What if there will be more printers? New package for every printer? Maybe something like package printers?